packages/components/src/internal/components/forms/BulkUpdateForm.tsx

```diff
@@ -13,18 +13,25 @@ import { capitalizeFirstChar, caseInsensitive, getCommonDataValues, getUpdatedDa
 
 import { QueryInfoForm } from './QueryInfoForm';
 
+type UpdateRows = (schemaQuery: SchemaQuery, rows: any[], comment?: string) => Promise<any>;
+type UpdateModel = (changes: any) => Promise<void>;
+
+function isUpdateModel(fn: UpdateRows | UpdateModel): fn is UpdateModel {
```

Comment:
Iffy on this, but wanted to enforce passing either UpdateRows or UpdateModel

Reply:
In general I think we should use type guards more often, so I'm in favor of this, however I am not sure I understand why checking against length works here, so we should probably put a comment in that explains why `fn.length === 1` means that the type is an Update model.